src/placeos-rest-api/controllers/metadata.cr

```diff
@@ -106,5 +120,10 @@ module PlaceOS::Api
       # Find will raise a 404 (not found) if there is an error
       @zone = Model::Zone.find!(id)
     end
+
+    def guest_zones
```

Comment:
this method is `guest_zones` yet it's querying for `ControlSystem`s

Reply:
yeah as they only have access to the zone_id that are in that system

Reply:
ahh i see

Reply:
Can we assume that the `sys_id` will always be the last value in `roles`?

Reply:
yeah the guest JWT is defined here: https://docs.google.com/document/d/1LyHi1oeuIDE2WIdAmACpNaLhsITB6aALYk52EYVWAVc/edit